manage/trigger.rst

```diff
@@ -1,112 +1,65 @@
 Trigger
 =======
 
-Every time a customer creates a new ticket, they automatically receive a
-confirmation email to assure them that their issue has been submitted
-successfully. This behavior is built into Zammad, but it’s also highly
-customizable, and you can set up other automated actions just like it.
+.. toctree::
+   :hidden:
 
-Maybe you want to set a higher priority on any ticket with the word “urgent” in
-the title. Maybe you want to *avoid* sending auto-reply emails to customers
-from certain organizations. Maybe you want mark a ticket as “pending” whenever
-someone adds an internal note to a ticket.
+   /manage/trigger/conditions-and-actions
+   /manage/trigger/examples
+   /manage/trigger/other-notifications
 
-Whatever it is, you can do it with **triggers:** actions that watch tickets for
-certain changes, and then fire off whenever those changes occur.
```

Comment:
Is there a reason we cut this intro down and moved what's left of it to the "Default Configuration" section? It's important, IMO.

I see two ways someone will arrive at this docs page:

1. **By referral.** Someone emails support and says, "Hi, I'd like Zammad to do X when Y happens. How can I set that up?" You respond, "With triggers, dumdum! Here, check out this link."
2. **By discovery/self-directed learning.** Someone signs up for Zammad and is trying to understand all the different parts of the system. They have no mental model for how everything works in Zammad, so none of the terms make sense to them. "What the heck is a trigger? Hm, well the docs page is probably a good place to start."

In the first case, the user _already knows what problem triggers solve,_ and just needs to know more about how they work. In the second case, the user doesn't have this context, and that's what the intro is for. It's small, but it's crucial—without it, the docs become very confusing. You're reading about _how something works_ without knowing _what it does._ When that happens to me, it quickly leads to despair.

Then again, you interact with customers and I don't. Would you say that case #2 is not realistic? Or that this intro is too long and distracting for case #1?

Curious to hear your thoughts.

Reply:
Honestly, I didn't see that I removed the introduction entirely.

My goal was to reduce duplicate text and to make it "less".

So, in my opinion, most of the introduction text moved into the "scope" section. Yes, I've changed wording a lot. I thought this would enhance irritations on these parts.

In the past weeks and months I for example saw a lot of confusion on 
* when do I use a trigger?
* when do I use a scheduler?

Users get confused by them, because especially for triggers they sometimes don't work as straight forward as you might think. This is why I've been trying to work out the scope a trigger can help you. (that's also the reason I came up with the heading - but I agree, it's miss leading. )

I agree for scope, we should change it back to "How do triggers work?" to assist readers better. I didn't think it fits so I changed it, imho a bad idea. :-D

Reply:
> I agree for scope, we should change it back to "How do triggers work?" to assist readers better. I didn't think it fits so I changed it, imho a bad idea. :-D

Actually, I vote "Limitations (and other ways to automate)" as a new name for "Scope". It took me a while to figure out what you were trying to say, but I do think it's different from "How do triggers work?" (I also see that it's important now!)

---

100% hear you on educating the user. Can't save everyone! ;)

I'm definitely learning to keep thing shorter. You know the pain of getting a user to read the docs, and I don't, but I think it's safe to say that long documentation doesn't get read! Mark Twain famously said,

> I didn't have time to write a short letter, so I wrote a long one instead.

So, that's something to think about.